Automattic/jetpack
# VideoPress: Add ContextualUpgradeTrigger to dashboard

Closes #26314

#### Changes proposed in this Pull Request:

* Adds `hasPaidPlan` to VideoPress initial state
* Adds a `ContextualUpgradeTrigger` component to the dashboard

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* WIth VideoPress plugin installed, but without a plan that supports VideoPress, go to the dashboard
* Check that the update trigger is shown
* With a plan that supports VideoPress, go to the dashboard
* Check that the update trigger does not render

![Screenshot_2022-09-23_16-19-17](https://user-images.githubusercontent.com/8486249/192044134-22f09107-7aa2-411a-9bb9-6fecea9e8c8f.png)
![Screenshot_2022-09-23_16-20-25](https://user-images.githubusercontent.com/8486249/192044173-e22b4e00-ea68-4f4d-9cff-01ed1166448c.png)
![Screenshot_2022-09-23_16-22-24](https://user-images.githubusercontent.com/8486249/192044196-21190166-ba9c-4e1e-9070-8c03875045f9.png)
